war_or_peace_runner.rb

```diff
@@ -0,0 +1,69 @@
+require 'pry'
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+require './lib/card_generator'
+
+class Game 
```

Comment:
This class should be in a file called `game.rb` and the only code that should be in this runner are what is necessary to start the game which I imagine would look something like this:

```ruby
require './lib/game'
Game.new
```

A runner file is just a jumping-off point, not a place to define classes and implementation code.

Moving beyond that though, I like the way that this class is set up, all your variables and methods are named appropriately and help to explain what responsibility each one has. Very easy on the eyes and easy to get through.